### DISCUSSION TITLE
Docker version don't work, desktop version how can I change IP and port?

### DISCUSSION THREAD

**Discussion by zinwelzl** [pablosnt/rekono#243] [Help]
Hi.

Kali latest

Docker version don't start web server, installation works, but web server restarting all the time?
docker stats and netstat show that.

For desktop version, how can I change IP and port?
Now it's 0.0.0.0:8000

**Comment by pablosnt** [comment]
Can you see some error message?

Rekono Desktop only run the backend, so this IP and port are right. After run it, the desktop application should be open (it could take some seconds, because it needs to do some things before open the app). But the web server won't be available at 443 port like when you run Rekono with docker-compose

**Comment by pablosnt** [comment]
I'm closing this discussion, please, reopen it if you need it. Thanks!
